week1/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>help hamsick</title>
+  </head>
+
+  <body>
+    <header>
+      <div class="headerColorBox">
+      <h1>💛🍔 햄식이를 도와줘 🍔💛</h1>
+      </div>
+    </header>
+
+    <main>
+      <aside class="sidebar">
+        <section class="option_card">
+          <h2 class="optionTitle">다른거 먹자</h2>
+          <div class="option"> 
+            <p class="macdonalds">맥도날드</p>
+            <p class="burgerking">버거킹</p>
+          </div>
+        </section>
+      </aside>
+      <section class="burger">
+        <article class="burger_card">
+          <img src="https://www.mcdonalds.co.kr/upload/product/pcfile/1583727855319.png" alt="빅맥사진">
```

Comment:
img테그의 alt 속성엔 "사진" 이나 "이미지"같은 워딩을 사용하지 않아요!
기본적으로 alt 속성을 적는 이유가 웹접근성을 위한 건데, img테그를 쓴 시점에서 이미 이 요소는 이미지임이 읽히기 때문에 alt속성 사진이라는 워딩을 넣으면 불필요하게 의미가 중복되게 됩니다 ! 
```suggestion
          <img src="https://www.mcdonalds.co.kr/upload/product/pcfile/1583727855319.png" alt="빅맥">
```

Reply:
진짜....대박대박.....필기해둘게요 담엔 꼬옥 써먹어야지....